docs/modules/deployment-launcher/launch-deployments.md

```diff
@@ -106,7 +106,7 @@ Where N is in range [1, number of simulated player deployments]
 
 ### Launch deployments
 
-At the bottom of the deployment configurations panel, there is a **Launch deployment** button. You can select which deployment configuration you wish to launch and then select **Launch deployment**.
+At the bottom of the deployment configurations panel, there is the **Launch deployment** button. You can select which deployment configuration you wish to launch and then select **Launch deployment**.
```

Comment:
As discussed. Seperate this into "In the deployment configurations panel, select the deployment configuration you wish to lauch", and describe where you can select this.
Then mention to select the button. There is no need to mention to location of this button or the panel, since the user is already in the right context.